.golangci.yml

```diff
@@ -3,20 +3,115 @@ linters:
   enable:
     - deadcode
     - gocyclo
+    - revive
     - gosec
     - govet
     - ineffassign
     - misspell
     - staticcheck
     - errcheck
+    - unused
     - gosimple
     - varcheck
 
 linters-settings:
   gofmt:
     simplify: false
-  goimports:
-    local-prefixes: github.com/cosmos/cosmos-sdk
+  gosec:
+    excludes:
+    - G404 # G404: Use of weak random number generator (math/rand instead of crypto/rand)
```

Comment:
is this exclusion required in our project?

Reply:
thought I saw this in some test cases before...seems ok without this